### ISSUE TITLE
Cannot map blob

### ISSUE THREAD

**Issue by davidkarlsen** [arnaudroger/SimpleFlatMapper#550]
I use sfm with jooq - sfm-jooq 4.0.1.
My table has a blob field, named "data", my pojo's (not records, but hand-made pojos) have a field
`
 byte[] blob
`

I have SfmRecordMapperProvider, provided through 

```
SfmRecordMapperProviderFactory.newInstance().
.addAlias("DATA", "blob")
                    .addColumnProperty("DATA", ConverterProperty.of(new Converter<Blob, byte[]>() {
                    @Override
                    public byte[] convert(Blob in) throws Exception {
                        return in.getBytes(0, Long.valueOf(in.length()).intValue());
                    }
                }))
```

but no way will sfm find a fitting property. I've tried with and without the addColumnProperty.

The error I get is

```
org.simpleflatmapper.map.MapperBuildingException: Could not find eligible property for 'blob' on  class com.edb.fs.tac.jfr.srv.model.filestore.LobData not found  See https://github.com/arnaudroger/SimpleFlatMapper/wiki/Errors_PROPERTY_NOT_FOUND
```

as if I had not provided any custom config

**Comment by arnaudroger** [comment]
will have a look thanks for the bug report

**Comment by arnaudroger** [comment]
what class are you mapping to ? com.edb.fs.tac.jfr.srv.model.filestore.LobData? looks like it's looking for blob property on the class. is there a setter or constructor injected?

**Comment by davidkarlsen** [comment]
i am mapping from a Record into LobData (my pojo). it has a byte[] setter.

**Comment by arnaudroger** [comment]
Could you send me the code for the LobData pojo? arnaud.roger at gmail.com

**Comment by davidkarlsen** [comment]
done

**Comment by arnaudroger** [comment]
nervermind just did, it dot work from me without the alias. because sfm recognized the @Column.
but the double alias somehow is confusing it will check

**Comment by arnaudroger** [comment]
yes looks like there is a conflict with the @column annotation and the alias, data -> blob -> data ....
could you try without the alias?

**Comment by davidkarlsen** [comment]
ah - yes - that worked.
I've been a bit back and forth with using annotations and having it all w/o annotations - but it seemed to work in one direction - but not the other one.

**Comment by arnaudroger** [comment]
The annotation change the prop name to the column, the prop change the column name to the prop... so when both present confusion ensue. I have a fix for it ready though. Nice catch!

**Comment by davidkarlsen** [comment]
BTW: I noticed jooq recently got an RecordUnmapper interface too https://www.jooq.org/javadoc/3.10.x/org/jooq/class-use/RecordUnmapper.html - that might be of interest to you/sfm?

**Comment by arnaudroger** [comment]
Seems doable will add a issue for that.

**Comment by arnaudroger** [comment]
so will release 2 fixes as part of 5.0.0. 
1 - stop the confusion when we get an alias from the annotation and property. it will ignore the annotation
2 - added a Blob to byte[] converter
